.github/workflows/multi-arch-ocp4-content.yml

```diff
@@ -0,0 +1,73 @@
+name: Multiarch build
+
+on:
+  push:
+      
+env:
+  IMAGE_NAME: compliance-operator-content
+  IMAGE_TAG: multi-arch
+  IMAGE_REGISTRY: quay.io
+  IMAGE_NAMESPACE: complianceascode
+
+jobs:
+  build:
+    name: Build and push multi-arch image
+    runs-on: ubuntu-20.04
+    
+    steps:
+      - name: add checkout action...
+        uses: actions/checkout@v2
+               
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+            
+      - name: Build image linux/amd64
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: ${{ env.IMAGE_NAME }}
+          tags: linux-amd64
```

Comment:
This is going to assume `linux-amd64` is the latest tag. Is that the intention? Also, this doesn't specify a push target (like master or a specific tag/branch).

In that case, it might be possible to tag a branch with the same tag as master, and vice versa. Is that how consumes are expecting this to work?

Reply:
Hi @rhmdnd
As per your suggestion I have updated code with changes.
Approach followed:
1. When code is pushed to master branch, it will build image with latest tag.
2. For newer release tag, image will be build with release tag.
For this I have created two separate workflows as per the PR https://github.com/ComplianceAsCode/compliance-operator/pull/83 which is merged.

So we will have two different files for content image. 

Compliance operator Content workflow files:
multi-arch_content_release-latest.yml
multi-arch_content_release.yml

Please let me know for any more changes.